src/components/generics/date-picker/date-picker.tsx

```diff
@@ -0,0 +1,82 @@
+import React from "react";
+import DateFnsUtils from "@date-io/date-fns";
+import { MuiPickersUtilsProvider, KeyboardDatePicker } from "@material-ui/pickers";
+import { Box } from "@material-ui/core";
+import strings from "localization/strings";
+import fiLocale from "date-fns/locale/fi";
+import enLocale from "date-fns/locale/en-US";
+import { useAppSelector } from "app/hooks";
+import { selectLocale } from "features/locale/locale-slice";
+import useDatePickerStyles from "styles/generics/date-picker/date-picker";
+
+/**
+ * Component properties
+ */
+interface Props {
+  dateFormat?: string;
+  selectedStartDate: Date | null;
+  onStartDateChange: (date: Date | null) => void;
+}
+
+/**
+ * Generic date picker component
+ */
+const GenericDatePicker: React.FC<Props> = ({
+  dateFormat,
+  selectedStartDate,
+  onStartDateChange
+}) => {
+  const classes = useDatePickerStyles();
+
+  const { locale } = useAppSelector(selectLocale);
+
+  const todayDate = new Date();
```

Comment:
Move this to KeyboardDatePicker render since this is not used anywhere else

Reply:
Done.